front/src/modules/infra/components/InfraSelector/InfraSelectorModalBodyStandard.tsx

```diff
@@ -105,27 +105,30 @@ const InfraSelectorModalBodyStandard = ({
               })}
               key={nextId()}
             >
-              <button
+              <div
                 className="infraslist-item-choice-main"
-                type="button"
                 onClick={() => {
                   setInfraID(infra.id);
                 }}
+                tabIndex={0}
+                role="button"
               >
-                <span className="infraslist-item-choice-name">{infra.name}</span>
-                {infra.locked && (
-                  <span className="infra-lock">
-                    <small>{t('infraManagement.locked')}</small>
-                    <Lock />
-                  </span>
-                )}
-              </button>
-              <GrantsManager
-                resourceId={infra.id}
-                resourceType="infra"
-                userGrant={userGrant!}
-                privilegesByGrant={grantTreeByResourceType.get('infra')}
-              />
+                <div className="infraslist-item-info-wrapper">
```

Comment:
nit: since we use the terms `main` and `footer` maybe we could use `header` instead of `wrapper` ? (just a personal opinion)
```suggestion
                <div className="infraslist-item-info-header">
```